app/src/main/kotlin/nl/q42/template/MainActivity.kt

```diff
@@ -26,7 +25,9 @@ class MainActivity : ComponentActivity() {
 
                 val navController = rememberNavController()
 
-                Surface(modifier = Modifier.fillMaxSize(), color = MaterialTheme.colorScheme.background) {
+                AppSurface (
+                    modifier = Modifier.fillMaxSize(),
+                ) {
```

Comment:
it was already mentioned by Izadi, but enableEdgeToEdge should be added. See Musemkaart app

Reply:
I'll create a separate PR for this